### DISCUSSION TITLE
How to see prebuilt file name in console insead of built one?

### DISCUSSION THREAD

**Discussion by alejo9010** [parcel-bundler/parcel#8709] [Q&A]
im using parcel to build my site, but each time i want to debug in the dev console from chrome, i get pointed to the built version of the js file instead of the original one, any clue? 
```
{
  "name": "movie-project",
  "version": "1.0.0",
  "description": "Movies project for learning",
  "default": "index.html",
  "scripts": {
    "start": "parcel index.html",
    "build": "parcel build index.html"
  },
  "author": "Ale",
  "license": "ISC",
  "devDependencies": {
    "parcel": "^2.8.2"
  }
}
```

**Comment by mischnic** [comment]
First of all, remove `"default": "index.html"`.

What exactly is the problem with the dev console?

**Reply by alejo9010** [reply]
when im debugging in chrome, my console will show the line with problems in the built version of the js file `index.e37f48ea.js:535` instead of sometthing like `controller.js:15`

**Reply by mischnic** [reply]
And you do you have sourcemaps enabled in devtools settings (open via F1)?
If yes, please share a complete code example (zip, github repo, ...)

**Reply by pupok90** [reply]
i have sourcemaps enabled, still getting built js in console, this is the source code [Link](https://github.com/pupok90/moviepractice)

**Reply by mischnic** [reply]
That works for me: 

![Bildschirm­foto 2022-12-23 um 22 16 59](https://user-images.githubusercontent.com/4586894/209406284-1d686867-89f8-4bd1-8c53-73ee167bea21.png)


(But these line numbers are slightly off right now because of this issue: https://github.com/swc-project/swc/issues/6694)
